fixbackend/notification/email/templates/trial_expired.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<h1>Your Fix trial has ended.</h1>
+
+<p>
+  We noticed you haven't bought a Fix subscription yet, and your trial has expired. Your account has been downgraded to
+  the Free plan, and you have lost access to the features available in the paid plans.
+
+  But don't worry, your account data is still there and you can easily get back on track by subscribing to one of our
+  plans.
+</p>
+
+<h2>Easy Steps to get a Fix subscription</h2>
+<ol>
+  <li>
+    <strong>Workspace Settings:</strong> <a href="https://app.fix.security">Login to Fix</a> and locate the Workspace
+    Settings icon at the bottom left of your screen.
+  </li>
+  <li>
+    <strong>Billing:</strong> Within the Workspace Settings, you'll find the "Billing" section. Here's where
+    can select a subscription plan that suits your needs.
+  </li>
+  <li>
+    <strong>Select the plan and add the payment method:</strong> Follow the instructions on the screen to select a plan
+    and add your payment method.
+  </li>
+
+</ol>
+
+<h2>What Comes Next?</h2>
+<p>
+  If you won't get a subscription, your account will be soon downgraded to the Free plan.
```

Comment:
The beginning says: your account has been downgraded to free plan. This sentence here is confusing.
```suggestion
```